### PR TITLE
US-1889327: Updated H2 headings.

### PR DESCRIPTION
**Pull request by poojapoojari**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by poojapoojari** [comment]
Hello @adegeo,

The 7th PR with 20 file edits is ready for technical review. Could you please review this PR?

Thank you!

File: dotnet-desktop-guide/framework/wpf/controls/how-to-use-the-image-element.md [hunk 1]
```diff
@@ -26,7 +28,7 @@ This example shows how to include images in an application by using the <xref:Sy
   
  [!code-xaml[ImageElementExample_snip#ImageSimpleExampleInlineMarkup](~/samples/snippets/csharp/VS_Snippets_Wpf/ImageElementExample_snip/CSharp/ImageSimpleExample.xaml#imagesimpleexampleinlinemarkup)]  
   
-## Example  
+## Code example to render an image 200 pixels wide
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/how-to-use-the-image-element.md:31]:
```suggestion
## Render an image
```

File: dotnet-desktop-guide/framework/wpf/controls/position-the-cursor-at-the-beginning-or-end-of-text.md [hunk 2]
```diff
@@ -14,18 +15,18 @@ ms.assetid: c771a0b8-c6b4-4240-aecd-a21d0ba51a2e
...
 This example shows how to position the cursor at the beginning or end of the text contents of a <xref:System.Windows.Controls.TextBox> control.  
   
-## Example  
+## Define a TextBox control
  The following [!INCLUDE[TLA#tla_xaml](../../../includes/tlasharptla-xaml-md.md)] code describes a <xref:System.Windows.Controls.TextBox> control and assigns it a Name.  
   
  [!code-xaml[TextBox_MiscCode#_MoveCursorXAML](~/samples/snippets/csharp/VS_Snippets_Wpf/TextBox_MiscCode/CSharp/Window1.xaml#_movecursorxaml)]  
   
-## Example  
+## Position the cursor at the beginning of the TextBox control
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/position-the-cursor-at-the-beginning-or-end-of-text.md:23]:
```suggestion
## Position the cursor at the beginning
```

File: dotnet-desktop-guide/framework/wpf/controls/position-the-cursor-at-the-beginning-or-end-of-text.md [hunk 3]
```diff
@@ -14,18 +15,18 @@ ms.assetid: c771a0b8-c6b4-4240-aecd-a21d0ba51a2e
...
  To position the cursor at the beginning of the contents of a <xref:System.Windows.Controls.TextBox> control, call the <xref:System.Windows.Controls.TextBox.Select%2A> method and specify the selection start position of 0, and a selection length of 0.  
   
  [!code-csharp[TextBox_MiscCode#_CursorToStart](~/samples/snippets/csharp/VS_Snippets_Wpf/TextBox_MiscCode/CSharp/Window1.xaml.cs#_cursortostart)]
  [!code-vb[TextBox_MiscCode#_CursorToStart](~/samples/snippets/visualbasic/VS_Snippets_Wpf/TextBox_MiscCode/VisualBasic/Window1.xaml.vb#_cursortostart)]  
   
-## Example  
+## Position the cursor at the end of the TextBox control
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/position-the-cursor-at-the-beginning-or-end-of-text.md:29]:
```suggestion
## Position the cursor at the end
```

File: dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-control-a-mediaelement-play-pause-stop-volume-and-speed.md [hunk 4]
```diff
@@ -15,15 +16,15 @@ ms.assetid: 6885a730-e054-4c16-8c1e-ffe17b1f7c32
 # How to: Control a MediaElement (Play, Pause, Stop, Volume, and Speed)
 The following example shows how to control playback of media using a <xref:System.Windows.Controls.MediaElement>. The example creates a simple media player that allows you to play, pause, stop, and skip back and forth in the media as well as adjust the volume and speed ratio.  
   
-## Example  
+## Create a UI
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-control-a-mediaelement-play-pause-stop-volume-and-speed.md:19]:
```suggestion
## Create the UI
```

File: dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-create-a-shape-using-a-streamgeometry.md [hunk 5]
```diff
@@ -12,20 +14,20 @@ ms.assetid: 08f7c8ce-074b-49cd-9aba-cc9592d4ee51
...
   
-## Example  
+## Use a StreamGeometry to define a triangle
  The next example uses a <xref:System.Windows.Media.StreamGeometry> to define a triangle in code. First, the example creates a <xref:System.Windows.Media.StreamGeometry>, then obtains a <xref:System.Windows.Media.StreamGeometryContext> and uses it to describe the triangle.  
   
  [!code-csharp[GeometriesMiscSnippets_procedural_snip#StreamGeometryTriangleExampleWholePage](~/samples/snippets/csharp/VS_Snippets_Wpf/GeometriesMiscSnippets_procedural_snip/CSharp/StreamGeometryTriangleExample.cs#streamgeometrytriangleexamplewholepage)]
  [!code-vb[GeometriesMiscSnippets_procedural_snip#StreamGeometryTriangleExampleWholePage](~/samples/snippets/visualbasic/VS_Snippets_Wpf/GeometriesMiscSnippets_procedural_snip/visualbasic/streamgeometrytriangleexample.vb#streamgeometrytriangleexamplewholepage)]  
   
-## Example  
+## Create a method
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-create-a-shape-using-a-streamgeometry.md:30]:
```suggestion
## Create a geometric shape in code
```

**Review by adegeo — APPROVED** [review]
Thanks!

**Pull request merged**